app/services/flow/flow_state_machine.rb

```diff
@@ -176,6 +176,9 @@ def analytics_properties
         step: current_step,
         step_count: current_flow_step_counts[current_step_name],
         analytics_id: @analytics_id,
+        irs_reproofing: current_user&.decorate&.reproof_for_irs?(
```

Comment:
`current_user` might not work for the hybrid document capture path (sending a link to a phone). In other places we'd use [`effective_user`](https://github.com/18F/identity-idp/blob/54045ef4218b90dd0875958ba962947e5deccbb3/app/controllers/concerns/effective_user.rb#L2-L5). Should we do the same here?

Reply:
Will there always be an effective user?  Or should we do something like current_user || effective_user?

Reply:
Cool. Let me update it, that sounds like a good approach.

Reply:
updated